kelsvilla/LegoMindstormsVSExtension
# Line highlighter input validation

Added validation for the colors sent to the line highlighter configuration webview. All valid CSS3 colors should be accepted and converted to `#RRGGBBAA` format when being sent back. There is now checkboxes to handle cases where the user has their colors set to `none`. If so, the checkbox will be unticked and the value of that color picker will be ignored when sending back to the extension.

# Testing:
- Go to your User settings while running the extension
- For any of the settings available in the webview (Background color, outline color, selection color, text color) try some of the following color formats: RGB(rgb(255, 255, 255)), RGBA(rgba(255, 255, 255, 1)), HSL (hsl(240, 100%, 75%)), HSLA (hsla(240, 25%, 50%, .5)), #RRGGBB (#8050c8), #RRGGBBAA (#8050c8ff), and CSS color names (red, blue, coral, etc.).
- Open the webview
- Verify that the colors shown in the webview match the color you put in the settings.

For the async notifications:
- Turn on TTS.
- Launch voice commands.
- Use some commands.
- Note that messages (in general) should not get cancelled while in progress. Some messages may be skipped if they are too old.
